e2etest/pkg/frr/container/container.go

```diff
@@ -19,20 +20,78 @@ const (
 	frrConfigDir = "config/frr"
 	// FRR routing image.
 	frrImage = "frrouting/frr:v7.5.1"
+	// Host network name.
+	hostNetwork = "host"
 	// FRR container mount destination path.
 	frrMountPath = "/etc/frr"
 )
 
+type Frr struct {
+	executor.Executor
+	Name           string
+	configDir      string
+	NeighborConfig config.NeighborConfig
+	RouterConfig   config.RouterConfig
+	Ipv4           string
+	Ipv6           string
+}
+
+// New creates a new FRR container on the host and returns the corresponding *Frr.
+// A situation where a non-nil container and an error are returned is possible.
+func New(name string, nc config.NeighborConfig, rc config.RouterConfig, network string, hostipv4 string, hostipv6 string) (*Frr, error) {
```

Comment:
maybe we can find a better name than new? when reading the test code it wasn't clear for me that this function is staring the container

Reply:
What do you suggest? For me it feels new container = start a container with additional stuff

Reply:
done